daemon/sdnotify.go

```diff
@@ -2,30 +2,33 @@
 package daemon
 
 import (
-	"errors"
 	"net"
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
-
 // SdNotify sends a message to the init daemon. It is common to ignore the error.
```

Comment:
@s-urbaniak In practice, your `ErrNotSupported` is something that apps should ignore. I prefer to keep the "error" only for real errors: that allows apps to write simpler code:

```
if _, err := SdNotify("READY=1"); err != nil {
  fmt.Printf(err)
  os.Exit(1)
}
```